dotnet/src/SemanticKernel/Memory/VolatileMemoryStore.cs

```diff
@@ -3,7 +3,6 @@
 using System.Collections.Generic;
 using System.Linq;
```

Comment:
Not sure what's going on here. I think a fork sync needs to be done. VolatileMemoryStore should not be changed for this PR

Reply:
Refresh should have this right.